jdbc-adapter/virtualschema-jdbc-adapter/src/test/java/com/exasol/adapter/jdbc/JdbcAdapterPropertiesTest.java

```diff
@@ -246,4 +247,15 @@ public void testNoneAsExceptionValue() throws AdapterException {
                 JdbcAdapterProperties.getExceptionHandlingMode(properties));
         JdbcAdapterProperties.checkPropertyConsistency(properties);
     }
+
+    @Test
+    public void getIgnoreErrorList() {
+        Map<String, String> properties = new HashMap<>();
+        properties.put("IGNORE_ERROR_LIST", "ERrror_foo, error_bar    ,  another_error");
+        List<String> expectedErrorList = new ArrayList<>();
+        expectedErrorList.add("ERRROR_FOO");
+        expectedErrorList.add("ERROR_BAR");
+        expectedErrorList.add("ANOTHER_ERROR");
+        assertEquals(expectedErrorList, JdbcAdapterProperties.getIgnoreErrorList(properties));
```

Comment:
This is a typical example of why I recommend using [Hamcrest matchers](http://hamcrest.org/JavaHamcrest/) over the vanilla JUnit matchers. Check http://hamcrest.org/JavaHamcrest/javadoc/1.3/org/hamcrest/collection/IsIterableContainingInAnyOrder.html#containsInAnyOrder(java.util.Collection)

Reply:
s.th. for a refactoring branch since widely used